client/src/components/Footer/index.tsx

```diff
@@ -1,21 +1,9 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 import Link from '../helpers/link';
 import './footer.css';
 
-const propTypes = {
-  children: PropTypes.any
-};
-
-const ColHeader = ({ children, ...other }) => (
-  <div className='col-header' {...other}>
-    {children}
-  </div>
-);
```

Comment:
I got rid of this because I could not figure out where it is being used in the entire repo. If anyone has any idea of why this should've been left as-is, I'd be glad to add it back

Reply:
We are not changing any logic. So I would avoid changing things that are unrelated to avoid any side effects and limit the scope of QA. 

You should revert these changes.

Reply:
Done